opba-auth-rest-impl/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+tpp:
+  loginUrl: http://localhost:8085/v1/psu/login
+  keyValidityDays: 7
+  jwsAlgo: RS256
+  privateKey: MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQC7wtbVBBDWlZSV7JsDgEHBTHBT0FH0YZh96WwFPyg88Iqsif1Xv38ZoGqcHhYLswyzFiNyEuZWblza1v05LX+pW4oU0Pi/ITX4biUQr5pFquSVV0qOBYOf0JCzxIDsKK1TiFxhz3CRFZ+1DXc0x5ZJASqKxOwVf+rCSKI3yB0Z76+AJUfRb0yiLvMW38iWkC79WJcyXXHkD8OGEvyJadbsipQxxDOGjzM0fa/vBzd8EeRcpC8fsxfH/d8WJ4oB7f26eW2dTY2wKahlgaKyjmmc7DITrPF8nRW39hH54DDbtaTyWCX6v3jg9iwcgYkB9AWLYYxzyR4n2VubGBXpizhrAgMBAAECggEAeR6Cvv7Ubdgn9V6f0G5X/H12qFplw6OBXxfIWsoXBtiUXCDXkFczgjB9FSBDUGC7/Hp3ewq+E1s2AuVUjCHRjEpElJEJJSIjUpKazly4w+AtfDjHGz5rXhiIf65vC+hhhtUb4uyRzMWKNJzvHNjzFTzvWH0XXfcaVSHNovYv754VjGzwtdhgj0w8QYRsCq4evBsRCl+POuJdKYv9BDYj3ADffVgDU0Txs8Q2lYvm16V0Ihgbcd6Dq4WblZY3Bkh0s9WpZVZdIy/R6bLWx1BSJwHiT267Oiq0U+REW4J584FryIrsqMTNJPjEbybEskniCujHxkGnRixO1BJZPvdIwQKBgQD1Zd4oNhO7SVX8H2+bYTEjKxZYSA2B6E+7pWGn1d4cOX2jjNhj1SQkLfWXQg4JFljTh9pL3K0moL8MkrLoYq+TTqEngMh7OJtEYySbec5ZSA6MM36VDYbHyNvjKHPrzwtzzbbYOqFXF8OZT5Eq9X+jQTmqcmGnYipmcWPRdK1J4wKBgQDD34Au37qsEM6f7VVm7zY/hXb+bZ1px6nkS+QuW2hp270brENq3LNtcA3oUX8yr/X8H9OEyO7+oevFcEkHRyDMce80VrO1UytXVQDo7nF6vepbPYy8u/rF/FLicrkLPoqQCDnL+NYVtnZo/oaXfw506VMBbU20+Yt3eUoxN2m92QKBgHyO3UmJ0VAh9DwJ9MtodNmF+EyNHhYPRdFYK45WNuwanlQSnI9n03VtwpepFpi0OBf+f+Y7t8PAmdiW6aqc1MOviIsVii1JWT5IGH+2PvAAbCnrxVeAyyXDJSFTGivaJtT72NA+SkB7tnOt+GBWxhemds+2SSz93cmryl+5mm/DAoGBAKafS+WUALI6oeq99DZaK6WhnXYWPSAs70L7pCIjNBFJEfemdLtE/4WLGumT2kZ+FRzi3gZI/7WwfGY6fIUqZqZEDZ5SWxFWrLnf36RGolYtlF0FYOF0b4FYghGMUglUrH6V4MS46zURQG1Uh538QYoGqcu+9amL1q5NiWWQo+0RAoGBAO+gk6lD3U1gUqAzSo3ri8gszk2Jopky+lIUy9y3ebsFsc3EetT7A/OgiSj4KZAgHKJiCesbq1OL7VeTGpsl0dqXPNW2Mh+xVVMJHGJ7ydZ8lvlNGQTNSvBwZwjJBsZMHCSgjdOGDF88NJKo2/Bjlhdg7tWp2qTkTVm7hTdYUY4+
+  publicKey: MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu8LW1QQQ1pWUleybA4BBwUxwU9BR9GGYfelsBT8oPPCKrIn9V79/GaBqnB4WC7MMsxYjchLmVm5c2tb9OS1/qVuKFND4vyE1+G4lEK+aRarklVdKjgWDn9CQs8SA7CitU4hcYc9wkRWftQ13NMeWSQEqisTsFX/qwkiiN8gdGe+vgCVH0W9Moi7zFt/IlpAu/ViXMl1x5A/DhhL8iWnW7IqUMcQzho8zNH2v7wc3fBHkXKQvH7MXx/3fFieKAe39unltnU2NsCmoZYGiso5pnOwyE6zxfJ0Vt/YR+eAw27Wk8lgl+r944PYsHIGJAfQFi2GMc8keJ9lbmxgV6Ys4awIDAQAB
+  signAlgo: RSA
```

Comment:
please follow '-' naming convention `signAlgo` is `sign-algo` for consistency

Reply:
done